docs/core/development/policy/release-process.rst

```diff
@@ -281,9 +282,10 @@ Announce
   - Twitter, if you feel like it
   - ``#twisted`` topic on IRC (you'll need ops)
 
-5. Merge the release branch into trunk, closing the release ticket at the same time.
-6. Close the release milestone (which should have no tickets in it).
-7. Open a milestone for the next release.
+5. Run ``python -m incremental Twisted --dev`` to add a `dev0` postfix.
```

Comment:
No, I think the dev0 should happen before the merge (but after the tag). Will add that, anyway.